src/index.html

```diff
@@ -3,14 +3,58 @@
 
 <head>
   <meta charset="UTF-8" />
+  <!--sync version number with console log in init.ts-->
+  <title>CB-Miro v0.2.4</title>
+  <style>
+    body {
+      text-align: center;
+      margin: auto;
+    }
+    .cb-logo {
+      margin: 4rem;
+    }
+    .header {
+      display: flex;
+      justify-content: center;
+    }
+    .header label {
+      align-self: center;
+    }
+    a {
+      text-decoration: none;
+      color: #06c;
+    }
+    a:hover {
+      color: rgb(2, 88, 174);
+    }
+  </style>
```

Comment:
Can we externalize the styles into a separate .css-File or would there be a problem with the hosting?

Reply:
Attempted in 37575c9fadbe2b3d6472cb386ad6dc6c6fdcbece but reversed in 960dcf3d33ea17a6a818e3f23c72751ffe232f56 since this did indeed pose a problem and not work.